.github/workflows/release.yml

```diff
@@ -24,8 +24,8 @@ jobs:
           go-version: 1.19.5
 
       - name: Run GoReleaser
-        # GoReleaser v4.2.0
-        uses: goreleaser/goreleaser-action@f82d6c1c344bcacabba2c841718984797f664a6b
+        # GoReleaser v2.5.0
+        uses: goreleaser/goreleaser-action@5e15885530fb01d81d1f24e8a6f54ebbd0fed7eb
```

Comment:
Should we have a separate commit for this (separation of concern and also to cherry pick)

Reply:
Do we want to squash these commits later?